.github/workflows/CI.yml

```diff
@@ -7,15 +12,15 @@ on:
 
 env:
   __TAUCMDR_PROGRESS_BARS__: "disabled"
-  __TAUCMDR_SYSTEM_PREFIX__: "~/taucmdr/system"
+  __TAUCMDR_SYSTEM_PREFIX__: "system"
   CI: "ON"
-  INSTALLDIR: "taucmdr-test"
+  INSTALLDIR: "taucmdr"
```

Comment:
I changed this to mimic what I use for local development: installing taucmdr to `${HOME}/taucmdr`. We can change this back someone things this is a bad idea, but I find it is easier to reason about things that go wrong during CI when it more closely matches my local setup.